data-prepper-plugins/avro-codecs/src/main/java/org/opensearch/dataprepper/plugins/codec/avro/AvroSchemaParserFromS3.java

```diff
@@ -44,7 +44,8 @@ private static String getS3SchemaObject(AvroOutputCodecConfig config) throws IOE
                 .key(config.getFileKey())
                 .build();
         ResponseInputStream<GetObjectResponse> s3Object = s3Client.getObject(getObjectRequest);
-        final Map<String, Object> stringObjectMap = objectMapper.readValue(s3Object, new TypeReference<>() {});
+        final Map<String, Object> stringObjectMap = objectMapper.readValue(s3Object, new TypeReference<>() {
+        });
```

Comment:
Was this change intentional to split into two lines?

Reply:
This is what my IntelliJ formatter produced.